research/src/pages/button.mdx

```diff
@@ -16,6 +16,20 @@ import ComponentCoverage from '../components/component-coverage'
 
 <Anatomy component="Button" />
 
+## Accessibility
+
+- Use a touch target area of `44px` by `44px`
+- For multiple buttons provided on the page that perform the same function, one of the targets should be `44px` by `44px`.
+- For custom buttons made using `<div>`, `<span>`, `<svg>`or `<img>`,
+  - `role="button"` must be applied.
+  - `tabindex` property should be set to "0", so that it could get tab focus.
+  - Custom buttons should be activated by both `Enter` and `SPACE` key.
```

Comment:
It probably needs to be clear on what activation means; on keydown or keyup or keypress? I believe keydown.

Reply:
👍 - technically *all* buttons should have this behavior. I think the important call out here is that custom buttons need them defined programmatically (`onKeydown`).

Reply:
One other minor note that I think would be good to call out here; if a keydown is handled, `preventDefault` should be called to prevent default browser behaviors. Otherwise, space might end up scrolling the browser window. I don't think `stopPropagation` should be called though.

I'm thinking that native buttons already do this, so this is more about custom buttons

Reply:
See https://github.com/WICG/open-ui/pull/14/files#r338290817